Sources/Present/Auth/SignUp/RegistrationPassword/ViewController/RegistrationPasswordViewController.swift

```diff
@@ -0,0 +1,144 @@
+import UIKit
+import RxSwift
+import RxCocoa
+
+final class RegistrationPasswordViewController: BaseVC<RegistrationPasswordViewModel> {
+    private let disposeBag = DisposeBag()
+    
+    private let passwordLabel = UILabel().then {
+        $0.text = "비밀번호"
+        $0.font = .systemFont(ofSize: 16, weight: .bold)
+    }
+    
+    private let inputPasswordTextfield = UITextField().then {
+        $0.addLeftPadding()
+        $0.placeholder = "8~16자리 영문, 숫자, 특수문자 조합"
+        $0.layer.borderColor = ChoiceAsset.Colors.grayMedium.color.cgColor
+        $0.layer.borderWidth = 1
+        $0.layer.cornerRadius = 8
+    }
+    
+    private let checkPasswordTextfield = UITextField().then {
+        $0.addLeftPadding()
+        $0.placeholder = "비밀번호 재입력"
+        $0.layer.borderColor = ChoiceAsset.Colors.grayMedium.color.cgColor
+        $0.layer.borderWidth = 1
+        $0.layer.cornerRadius = 8
+    }
+    
+    private lazy var nextButton = UIButton().then {
+        $0.setTitle("다음", for: .normal)
+        $0.isEnabled = false
+        $0.backgroundColor = ChoiceAsset.Colors.grayVoteButton.color
+        $0.layer.cornerRadius = 8
+    }
+    
+    private let warningLabel = UILabel().then {
+        $0.font = .systemFont(ofSize: 14, weight: .semibold)
+        $0.isHidden = true
+        $0.textColor = .init(red: 1, green: 0.363, blue: 0.363, alpha: 1)
+    }
+    
+    private func showWarningLabel(warning: String) {
+        DispatchQueue.main.async {
+            self.warningLabel.isHidden = false
+            self.warningLabel.text = warning
+        }
+    }
+    
+    private func bindUI() {
+        let passwordObservable = inputPasswordTextfield.rx.text.orEmpty
+        let checkPasswordObservable = checkPasswordTextfield.rx.text.orEmpty
+        
+        Observable.combineLatest(
+            passwordObservable,
+            checkPasswordObservable,
+            resultSelector: { s1, s2 in (s1.count >= 8 && s1.count <= 16) && (s2.count >= 8 && s2.count <= 16) }
```

Comment:
이 코드를 **범위 연산자**로 변경하는건 어떨까요?
현재 코드는 두 개의 비교연산을 수행합니다. 만약 범위연산자를 사용하면 범위에 대한 체크를 한 번만 수행하기 때문에 훨씬 효율적입니다. 또한 ```contains``` 메서드는 내부적으로도 최적화 되어있기 때문에 성능상의 이점과 가독성 부분에서의 이점이 크다고 생각됩니다.
따라서 아래처럼 바꾸는건 어떨까요?

```suggestion
            resultSelector: { s1, s2 in (8...16).contains(s1.count) && (8...16).contains(s2.count) }
```

Reply:
저번 회원가입 구현 할 때 contains를 사용했었는데 이번에 사용을 안 했네요 .. ㅎㅎ 
피드백 감사합니다 😁